winpcapy/winpcapy.py

```diff
@@ -109,6 +109,17 @@ def run(self, callback=None, limit=0):
         self._callback = callback
         # Run loop with callback wrapper
         wtypes.pcap_loop(self._handle, limit, self._callback_wrapper, None)
+    def send(self, buf):
+        """
+        :send a bufer to the interface
+        """
+        assert self._handle is not None
+        l = (ctypes.c_int * 1)()
+        l = len(buf)
+        b = (ctypes.c_ubyte * l)()
```

Comment:
Perhaps consider other methods of buffer initialization

- b = (ctypes.c_char_p)(buf)  # work directly with char* if we can assume the buffer won't change (need to check with WinPcap)
- b = ctypes.cast( ctypes.create_string_buffer(buf,l), ctypes.POINTER(ctypes.c_ubyte)) # copy using create_string_buffer and cast to u_byte*